webapp/channels/src/packages/mattermost-redux/src/selectors/entities/files.ts

```diff
@@ -46,6 +47,18 @@ export function makeGetFilesForPost(): (state: GlobalState, postId: string) => F
     );
 }
 
+export function makeGetFilesForEditHistory(): (state: GlobalState, editHistoryPost: Post) => FileInfo[] {
```

Comment:
Would be nice to have tests for these. Specially consistency tests, so same input return same output.